content/blog/2024/10/2024-10-15-security-disclosure-matrix-js-sdk-and-matrix-react-sdk.md

```diff
@@ -0,0 +1,34 @@
++++
+title = "Security disclosure for matrix-js-sdk (CVE-2024-47080) and matrix-react-sdk (CVE-2024-47824)"
+date = "2024-10-15T11:39:35Z"
+path = "/blog/2024/10/2024-10-15-security-disclosure-matrix-js-sdk-and-matrix-react-sdk"
```

Comment:
```suggestion
```

path is only specified for TWIM to avoid clashes. No need to specify it for regular blog posts

Reply:
I was wondering what that was about as I didn't remember seeing it before. Thanks.